internal/policies/mount/mount_test.go

```diff
@@ -0,0 +1,106 @@
+package mount
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/adsys/internal/policies/entry"
+)
+
+func TestApplyPolicy(t *testing.T) {
+	t.Parallel()
+
+	entries := GetEntries(5, 0, 1, 0, []string{"\n"})
+
+	tests := map[string]struct {
+		entries    []entry.Entry
+		objectName string
+		computer   bool
+		uid        string
```

Comment:
I saw that on this one, you got inspired by the script part, which is great :)

However, I suggest that you keep the same name for the familiarity when reading tests between packages. This way, you can see things are similar, This includes:
- a blank line between the arguments of the `ApplyPolicy` function and the side effects, like returned uid/gid.
- `userReturnedUID` instead of `uid`, to clearly say this is not an input, but that we are mocking some side-effect returned value.